frame/election-provider-multi-phase/src/helpers.rs

```diff
@@ -33,7 +39,14 @@ macro_rules! log {
 	};
 }
 
-// This is only useful for a context where a `<T: Config>` is not in scope.
+/// Macro to log messages without including the system block number in the log.
+///
+/// This macro is a logging utility used within the crate. It allows logging messages without
```

Comment:
Please keep the principle of remaining concise and DRY in mind. 

This whole block should be: 

```
/// Same as [`log`], but in cases where a `T` implementing [`Config`] is not in scope. 
```